src/main/java/emissary/util/ShortNameComparator.java

```diff
@@ -19,34 +19,115 @@
     public int compare(IBaseDataObject obj1, IBaseDataObject obj2) {
         String s1 = obj1.shortName();
         String s2 = obj2.shortName();
```

Comment:
1. Does this new implementation return the same result as before if neither s1 nor s2 contain Family.SEP?
2. Does this new implementation return the same result as before if Family.SEP and s2 is "s1 + Family.SEP + [more]"?
3. Does this new implementation return the same result as before if Family.SEP and s1 is "s2 + Family.SEP + [more]"? 
4. Do we have test coverage for the above 3 scenarios to insure parity with previous behavior?